File: Praktikum Modul 2/atk.php
<?php
class ATK {
    public $jenisBarang = 3;
    public $nama = [];
    public $stok = [];
    public $hrgSatuan = [];
    public $hrgStok = [];
    public $hrgTotal = 0;

    public function inputBarang() {
        echo "Masukkan Data Barang ATK:\n";
        for ($i = 0; $i < $this->jenisBarang; $i++) {
            echo "Nama Barang: ";
            $this->nama[$i] = trim(fgets(STDIN));

            echo "Jumlah Stok: ";
            $this->stok[$i] = (int) trim(fgets(STDIN));

            echo "Harga Satuan: ";
            $this->hrgSatuan[$i] = (int) trim(fgets(STDIN));

            echo "______________________\n";
        }
    }

    public function tampilBarang() {
        echo "Data Harga Barang ATK:\n";
        for ($i = 0; $i < $this->jenisBarang; $i++) {
            echo "Nama Barang: " . $this->nama[$i] . "\n";
            echo "Jumlah Stok: " . $this->stok[$i] . "\n";
            echo "Harga Satuan: Rp " . number_format($this->hrgSatuan[$i], 0, ',', '.') . "\n";

            $this->hrgStok[$i] = $this->hrgSatuan[$i] * $this->stok[$i];
            echo "Harga Keseluruhan: Rp " . number_format($this->hrgStok[$i], 0, ',', '.') . "\n";
            echo "______________________\n";
        }
    }

    public function hargaTotal() {
        $this->hrgTotal = array_sum($this->hrgStok);
        echo "Harga Total Seluruh Barang: Rp " . number_format($this->hrgTotal, 0, ',', '.') . "\n";
        echo "______________________\n";
    }
}


$obj = new ATK();
$obj->inputBarang();
$obj->tampilBarang();
$obj->hargaTotal();
?>
`

File: Praktikum Modul 2/atk.java
/*
 Seorang penjual alat tulis menjual 10 bolpoint, 10 pensil dan 10 penghapus. 1 biji bolpoint
 harganya Rp. 2000, 1 biji pensil harganya Rp. 1.000 dan 1 penghapus harganya Rp. 500.
 Gunakanlah objek untuk menyelesaikan soal dibawah ini!
 
 1. Buatlah method untuk memasukkan (setter) nama, stok, harga satuan, dan harga (stok x
 harga satuan) alat tulis tersebut! (40 point)

 2. Buatlah method untuk menampilkan (getter) nama, stok, harga satuan, dan harga (stok x
 harga satuan) alat tulis tersebut! (40 point)

 3. Buatlah method Total Harga (setter getter) untuk menampilkan uang yang diterima penjual
 jika semua alat tulis tersebut terjual semuanya! (20 point)

*/ 
import java.util.Scanner;
public class atk {
    int jenisbarang = 3;
    int stok[] = new int[jenisbarang];
    int hrgsatuan[] = new int[jenisbarang];
    int hrgstok[] = new int[jenisbarang];
    int hrgtotal;
    String[] nama = new String[jenisbarang];
    Scanner inputdata = new Scanner(System.in);

    public void inputbarang(){
        System.out.println("Masukkan Data Barang ATK");
        for(int i=0; i<3; i++){
            System.out.print("Nama Barang: ");
            nama[i]= inputdata.next();
            System.out.print("Jumlah Stok: ");
            stok[i] = inputdata.nextInt();
            System.out.print("Harga Satuan: ");
            hrgsatuan[i] = inputdata.nextInt();
            System.out.println("______________________");
           }
        
    }
    public void tampilbarang(){
        System.out.println("Data Harga Barang ATK");
        for(int i=0; i<3; i++){
            System.out.print("Nama Barang: ");
            System.out.println(nama[i]);
            System.out.print("Jumlah Stok: ");
            System.out.println(stok[i]);
            System.out.print("Harga Satuan: ");
            System.out.println(hrgsatuan[i]);
            System.out.print("Harga Keseluruhan: ");
            hrgstok[i] = hrgsatuan[i]*stok[i];
            System.out.println(hrgstok[i]);
            System.out.println("______________________");
        }
        
    }
    public void hargatotal(){
        hrgtotal=hrgstok[0]+hrgstok[1]+hrgstok[2];
        System.out.print("Harga Total Seluruh Barang : ");
        System.out.println(hrgtotal);
        System.out.println("______________________");

    }
    public static void main(String[] args) {
    atk obj = new atk();
    obj.inputbarang();
    obj.tampilbarang();
    obj.hargatotal();
    }
}